src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -463,6 +463,35 @@ def _llm_token_counts(usage: "CompletionUsage") -> Iterator[tuple[str, Any]]:
         yield LLM_TOKEN_COUNT_COMPLETION, usage.completion_tokens
         yield LLM_TOKEN_COUNT_TOTAL, usage.total_tokens
 
+        if hasattr(usage, "prompt_tokens_details") and usage.prompt_tokens_details is not None:
+            prompt_details = usage.prompt_tokens_details
+            if (
+                hasattr(prompt_details, "cached_tokens")
+                and prompt_details.cached_tokens is not None
+            ):
+                yield LLM_TOKEN_COUNT_PROMPT_DETAILS_CACHE_READ, prompt_details.cached_tokens
+            if hasattr(prompt_details, "audio_tokens") and prompt_details.audio_tokens is not None:
+                yield LLM_TOKEN_COUNT_PROMPT_DETAILS_AUDIO, prompt_details.audio_tokens
```

Comment:
just checking we have a min version pinned for these

Reply:
just verified:

we require: `openinference-semantic-conventions>=0.1.20`
audio requires: `0.1.17`
cache and reasoning tokens require: `0.1.16`